src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
```

Comment:
1. Rethink naming - you create instance of Lottery and name it ball, you think it's right?
2. Do you really need 3 instances of Lottery in this case?